src/android/CameraLauncher.java

```diff
@@ -1312,20 +1312,39 @@ public void onScanCompleted(String path, Uri uri) {
     }
 
     public void onRequestPermissionResult(int requestCode, String[] permissions,
-                                          int[] grantResults) {
-        for (int r : grantResults) {
-            if (r == PackageManager.PERMISSION_DENIED) {
-                this.callbackContext.sendPluginResult(new PluginResult(PluginResult.Status.ERROR, PERMISSION_DENIED_ERROR));
-                return;
+                                          int[] grantResults) throws JSONException {
+        String version = android.os.Build.VERSION.RELEASE;
+        if(Integer.parseInt(version) == 13)
+        {
+            if(PermissionHelper.hasPermission(this, Manifest.permission.CAMERA)) {
+                switch (requestCode) {
+                    case TAKE_PIC_SEC:
+                        takePicture(this.destType, this.encodingType);
+                        break;
+                    case SAVE_TO_ALBUM_SEC:
+                        this.getImage(this.srcType, this.destType, this.encodingType);
```

Comment:
question: why you have 3 params now? old code was `this.getImage(this.srcType, this.destType);` same question below row 1345

Reply:
Good catch and will trigger a compiler error in this PR's current state:

```
error: method getImage in class CameraLauncher cannot be applied to given types;
                        this.getImage(this.srcType, this.destType, this.encodingType);
                            ^
  required: int,int
  found: int,int,int
  reason: actual and formal argument lists differ in length
```

Reply:
yes I will check